Tests/IntegrationTests/KafkaTests.swift

```diff
@@ -1,21 +1,21 @@
 //===----------------------------------------------------------------------===//
 //
-// This source file is part of the swift-kafka-gsoc open source project
+// This source file is part of the swift-kafka-client open source project
 //
-// Copyright (c) 2022 Apple Inc. and the swift-kafka-gsoc project authors
+// Copyright (c) 2022 Apple Inc. and the swift-kafka-client project authors
 // Licensed under Apache License v2.0
 //
 // See LICENSE.txt for license information
-// See CONTRIBUTORS.txt for the list of swift-kafka-gsoc project authors
+// See CONTRIBUTORS.txt for the list of swift-kafka-client project authors
 //
 // SPDX-License-Identifier: Apache-2.0
 //
 //===----------------------------------------------------------------------===//
 
 import struct Foundation.UUID
+@testable import Kafka
```

Comment:
Can we check in a follow up if we can get rid of all the `@testable` imports and just test the public API.